src/test/scala/com/microsoft/hyperspace/index/rankers/FilterIndexRankerTest.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index.rankers
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.catalyst.expressions.AttributeReference
+import org.apache.spark.sql.types.{IntegerType, StringType}
+
+import com.microsoft.hyperspace.index.FileInfo
+
+class FilterIndexRankerTest extends SparkFunSuite {
+
+  val t1c1 = AttributeReference("t1c1", IntegerType)()
+  val t1c2 = AttributeReference("t1c2", StringType)()
+  val t2c1 = AttributeReference("t2c1", IntegerType)()
+  val t2c2 = AttributeReference("t2c2", StringType)()
+
+  test("testRank: temporal default behavior - return head") {
+    val ind1 = RankerTestHelper.index("ind1", Seq(t1c1), Seq(t1c2))
+    val ind2 = RankerTestHelper.index("ind2", Seq(t1c1), Seq(t1c2))
+    val ind3 = RankerTestHelper.index("ind3", Seq(t2c1), Seq(t2c2))
+
+    val indexes = Seq(ind1, ind2, ind3)
+    assert(FilterIndexRanker.rank(indexes, false).get.equals(ind1))
+  }
+
+  test("testRank: return the index with the largest source file list if HybridScan is enabled") {
```

Comment:
nit: Why `testRank` instead of `FilterIndexRankerTest`? Sorry for the naive question - I'm trying to understand how we should name these tests.

Reply:
I just used the previous one in JoinRankerTest. Yes we need some guideline for the naming 🤔

Reply:
@imback82 Any recommendations?

Reply:
Yea, `testRank` seems weird. I would do something like `test("rank() should return the index with the largest number of source files if HybridScan is enabled)`. (Note that `FilterIndexRanker` is omitted since it's already under `FilterIndexRankerTest`).

Reply:
Oh... good you wrote a great name 🙂

I was also hoping: can we agree on a naming convention for tests now that the code base complexity is increasing? For instance:

  1. Tests do not need to contain the test method name
  2. Name of the test should clearly state the hypothesis and what call out what is being tested
  3. ...